src/components/NavBar/SearchBar.css.ts

```diff
@@ -129,11 +128,11 @@ export const searchBarDropdown = style([
   sprinkles({
     borderBottomLeftRadius: '12',
     borderBottomRightRadius: '12',
-    background: 'backgroundSurface',
     height: { sm: 'viewHeight', md: 'auto' },
   }),
   {
     borderTop: 'none',
+    backgroundColor: vars.color.backgroundSurface,
```

Comment:
this previous syntax of `background: 'backgroundSurface'` in sprinkles was correct and should be used here. You can also just change `background` to` backgroundColor`